README.md

```diff
@@ -38,3 +38,9 @@ cluster set in kube config.
 | kube_summary_pod_ephemeral_storage_inodes_free     | Number of available Inodes for pod Ephemeral storage                 | pod, namespace       |
 | kube_summary_pod_ephemeral_storage_inodes          | Number of Inodes for pod Ephemeral storage                           | pod, namespace       |
 | kube_summary_pod_ephemeral_storage_inodes_used     | Number of used Inodes for pod Ephemeral storage                      | pod, namespace       |
+| node_runtime_imagefs_available_bytes               | Number of bytes of node Runtime ImageFS that aren't consumed         | node                 |
```

Comment:
Thanks @caruccio !

Only comment I have - can we prefix the new metrics names with `kube_summary_` please - I think that will keep it more consistent with others.

Then I'll merge + create a new release.

Thanks!

Reply:
Oh yeah! My bad. Just updated it and push-force into my branch. Thanks for noticing.